Task4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# Пример:
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+s = int(input())
+print((s // 6), ((s // 6) * 4), (s // 6))
```

Comment:
В Python скобки и другие подобные синтаксические конструкции используются куда реже, чем в других языках.
```suggestion
print(s // 6, s // 6 * 4, s // 6)
```
К тому же, обратите внимание, что у вас трижды используются одна и та же операция, а значит, на каждую эту операцию будут выделяться ресурсы. Чтобы такого не происходило использовать переменные.
```suggestion
var = s // 6
print(var , var  * 4, var)
```